### PR TITLE
To do list app

### PR DESCRIPTION
**Pull request by BertrandConxy**
# To do list app
In this branch, I have developed the to-do list app using React.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+## Screenshoots
+![Screenshot (28)](https://user-images.githubusercontent.com/90222110/155112486-9ef8f556-e195-4f94-a9a9-189161fdfe30.png)
+
+
+
+## Live Demo link
+
+
+
+## Issues
```
Comment (zairdon20) [inline README.md]: - [ ] Well done👏👏, You've  done a great job 👍👍 on the Readme.file, its very descriptive but there is one more thing you might want to add that you left out. You didn't add the live demo link after deploying the page, this is so important as it helps other's to see what you've done without cloning the repository on the local machine.

**Review by zairdon20 — CHANGES_REQUESTED** [review]
You've done tremendous work on the project so far 👏👏👏👏, but there are just few changes that you need to address before moving on to the next project.



image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md (hunk 1 above)
Comment (BertrandConxy) [reply README.md]: Yeah, I understand it. I am going to deploy it. Thank you.

**Comment by BertrandConxy** [comment]
https://todo250.netlify.app/

**Pull request merged**